### PR TITLE
Fix typo: seqence -> sequence

### PR DESCRIPTION
**Pull request by Mithi83**
Fix a typo in GRE and GRE_PPTP where "sequence" was spelled "seqence".

### CODE REVIEW

File: scapy/layers/l2.py [hunk 1]
```diff
@@ -474,7 +474,7 @@ class GRE(Packet):
                    ConditionalField(XShortField("chksum", None), lambda pkt:pkt.chksum_present == 1 or pkt.routing_present == 1),  # noqa: E501
                    ConditionalField(XShortField("offset", None), lambda pkt:pkt.chksum_present == 1 or pkt.routing_present == 1),  # noqa: E501
                    ConditionalField(XIntField("key", None), lambda pkt:pkt.key_present == 1),  # noqa: E501
-                   ConditionalField(XIntField("seqence_number", None), lambda pkt:pkt.seqnum_present == 1),  # noqa: E501
```
Comment (guedou) [inline scapy/layers/l2.py]: Please use the `deprecated_fields` functionality as shown here https://github.com/secdev/scapy/blob/master/scapy/layers/dot11.py#L244. 

Also, could you rename the fields in the unit tests https://github.com/secdev/scapy/blob/master/test/pptp.uts#L44 ?

**Comment by Mithi83** [comment]
I'll look into the deprecated_fields issue. I What would be the correct version number for the change? Another question: as I'm quite new to github and contributing to other projects: Would you like to get a second commit with the fix in my forked repo or would you like me to amend my original commit and force-push to my forked repo?

As for the second request: I'm pretty sure my original PR already contained the changes you asked for in https://github.com/secdev/scapy/blob/master/test/pptp.uts#L44 . Please check again and tell me if you need anything on top.

**Comment by guedou** [comment]
You can use 2.4.4 (the next Scapy realease).

No problem if you force push your commit.

I did not see your UTS commit. Sorry!

**Comment by Mithi83** [comment]
Should be fixed now.

**Comment by guedou** [comment]
Could you rebase your commit against master ? That will likely fix the failing tests.

**Comment by Mithi83** [comment]
I did and indeed most were successful, but not all. The failed items don't seem to be related to my changes. Please tell me if I need to rebase again.

**Comment by guedou** [comment]
@Mithi83 merged! It seems that a test failed due to a temporary error.

**Comment by Mithi83** [comment]
Sorry to bother you again. Although you wrote that it was merged I can't find it in the master branch. Am I looking in the wrong place or did you not merge after all?

**Comment by guedou** [comment]
@Mithi83 I should have made a mistake. I reopened the PR, and will merge it when all tests will pass.

**Pull request merged**